packages/babel-generator/test/index.js

```diff
@@ -259,14 +249,10 @@ describe("generation", function() {
       sourceMaps: true,
     });
 
-    chai.expect(generated.rawMappings).to.be.an("array");
-
-    chai
-      .expect(generated)
-      .ownPropertyDescriptor("map")
-      .not.to.have.property("value");
```

Comment:
I guess that this test asserts that `generated.map` is a getter, since `Object.getOwnPropertyDescriptor(generated.map)` shouldn't have `value`?

Reply:
Oh! It seems I misunderstood. But I'm still unable to figure out a proper test equivalent in Jest expect world. Any suggestions?

Reply:
@nicolo-ribaudo I feel the new test in Jest world still does test for the scenario you described (at least partly, where I don't think we don't have notion of propertyDescriptor in jest, but otherwise we are still covering everything).

Reply:
It should be `expect(Object.getOwnPropertyDescriptor(generated, "map")).not.toHaveProperty("value")` instead `expect(generated.map).not.toHaveProperty("value")`

Reply:
I don't know but this actually fails. 

Object.getOwnPropertyDescriptor(generated, "map") returns:
```
{
        value: {
           version: 3,
           sources: [ 'a.js' ],
           names: [ 'hi', 'msg', 'console', 'log' ],
           mappings: 'AAAA,SAASA,EAAT,CAAaC,GAAb,EAAkB;AAAEC,UAAQC,GAAR,CAAYF,GAAZ;AAAmB'
        },
        writable: true,
        enumerable: true,
        configurable: true
}
```

while generated.map is:
```
{
        version: 3,
        sources: [ 'a.js' ],
        names: [ 'hi', 'msg', 'console', 'log' ],
        mappings: 'AAAA,SAASA,EAAT,CAAaC,GAAb,EAAkB;AAAEC,UAAQC,GAAR,CAAYF,GAAZ;AAAmB'
}
```